data/lang/input-core/en.json

```diff
@@ -131,6 +131,22 @@
     "description": "Descriptive name for the PrimaryFire action.",
     "message": "Primary Fire"
   },
+  "BIND_RADAR_TOGGLE_MODE": {
+    "description": "Descriptive name for the RadarToggle action.",
+    "message": "Toggle Radar between 2D and 3D modes"
```

Comment:
```suggestion
    "message": "Toggle Radar Display Mode"
```

Reply:
Fixed in 556f767fed6741ec1621a0cd96527f9f321b9d32